samples/bluetooth/hci_uart/src/main.c

```diff
@@ -357,6 +357,32 @@ void main(void)
 
 	/* Enable the raw interface, this will in turn open the HCI driver */
 	bt_enable_raw(&rx_queue);
+
+	if (IS_ENABLED(CONFIG_BT_WAIT_NOP)) {
+		/* Issue a Command Complete with NOP */
+		int i;
+
+		const struct {
+			const u8_t h4;
+			const struct bt_hci_evt_hdr hdr;
+			const struct bt_hci_evt_cmd_complete cc;
+		} __packed cc_evt = {
+			.h4 = H4_EVT,
+			.hdr = {
+				.evt = BT_HCI_EVT_CMD_COMPLETE,
+				.len = sizeof(struct bt_hci_evt_cmd_complete),
+			},
+			.cc = {
+				.ncmd = 1,
+				.opcode = BT_OP_NOP,
```

Comment:
I know that it makes no difference in practice, however this should really be sys_cpu_to_le16(BT_OP_NOP) to keep track of what endianness `.opcode` is.

Reply:
Is this actually allowed on an initializer?

Reply:
On big endian this maps to `__bswap_16()` which is defined as follows:

```
#define __bswap_16(x) ((u16_t) ((((x) >> 8) & 0xff) | (((x) & 0xff) << 8))) 
```

I'd think the compiler should be able to handle that even for the initialiser of a const variable, but you could always try it by using `__bswap_16(BT_OP_NOP)`

Reply:
It is. Updated, thanks!